.github/workflows/frontend-verify.yml

```diff
@@ -143,11 +95,11 @@ jobs:
         run: docker logs connect
       - name: Run Playwright tests
         working-directory: frontend
-        run: npm run e2e-test
+        run: bun run e2e-test
       - name: Upload Playwright report artifact
         uses: actions/upload-artifact@v4
         if: ${{ failure() }}
         with:
           name: playwright-report
           path: frontend/playwright-report/
-          retention-days: 1
+          retention-days: 2
```

Comment:
I think 2 days is reasonable because we can check what may have failed over weekend